index.html

```diff
@@ -460,6 +460,76 @@ <h4>Definitions</h4>
       </section>
     </section>
 
+    <section>
+      <h2><dfn data-lt="created" data-dfn-type="dfn" id="created">created</dfn></h2>
+      <a href="https://www.w3.org/TR/did-core/#created">Created.</a>
```

Comment:
Concerned that this is a single word description. Can we elaborate into a sentence.